examples/Overhead.hs

```diff
@@ -12,7 +12,11 @@ import GHC.Stats as GHC
 main :: IO ()
 main = do
   M.initializeTime -- Need to do this before calling M.getTime
+#if defined(__GHCJS__)
+  statsEnabled <- return False
```

Comment:
Instead of adding CPP here, change the definition of the `getRTSStatsEnabled` backwards-compatibility shim on line 36 of this file.

(Also, if there is a GHCJS-specific reason why this must return `False`, it would be good to leave a comment explaining why.)

Reply:
done